infinispan-remote/opentelemetry/docker-compose/docker-compose.yaml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+
+networks:
+  mynetwork:
+    driver: bridge
+services:
+  jaeger:
+    image: jaegertracing/all-in-one:1.57
+    ports:
+      - "16686:16686"
+      - "4318:4318"
+    environment:
+      - LOG_LEVEL=debug
+    networks:
+      - mynetwork
+  infinispan-tracing:
+    image: quay.io/infinispan/server:15.0
+    ports:
+      - "11222:11222"
+    container_name: infinispan-tracing
+    environment:
+      USER: admin
+      PASS: password
+    networks:
+      - mynetwork
+    command: -c infinispan.xml -Dinfinispan.tracing.enabled=true -Dotel.traces.exporter=otlp -Dotel.service.name=my-infinispan-server -Dotel.exporter.otlp.endpoint=http://otlp-collector-host:4318
```

Comment:
@karesti with Infinispan 15 we evolved the way with which tracing is configured.
See https://infinispan.org/docs/stable/titles/server/server.html#opentelemetry-tracing.
Or https://github.com/fax4ever/infinispan-demo/blob/main/infinispan-15-tracing/infinispan.yaml